notebooks/en/agent_text_to_sql.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<p>Add a link to <code>transformers.agents</code> so curious readers can learn more.</p><p> </p><p>Keep formatting consistent for how we refer to text-to-SQL:</p><p> </p><p>"What's the advantage over a classical text-to-SQL pipeline?"</p><p> </p><p>"A classical text-to-SQL pipeline..."</p><p> </p><p>"An agent system is able to critically inspect outputs and decide if the query needs to be changed or not..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>